themes/default/content/blog/kubernetes-server-side-apply/index.md

```diff
@@ -0,0 +1,748 @@
+---
+title: "Manage Shared Kubernetes Resources Safely with Pulumi"
+
+date: 2022-07-27T00:00:00-00:00
+
+meta_desc: Manage shared Kubernetes resources safely with our new Server-Side Apply support
+meta_image: meta.png
+authors:
+    - levi-blackstone
+tags:
+    - kubernetes
+---
+
```

Comment:
A few minor notes on narrative structure:
1. We could potentially add a sentence at the beginning just restating what the Pulumi Kubernetes provider is and some context for this - "We first launched the Pulumi Kubernetes provider in 2018, and it has been one of the most used providers across the entire Pulumi ecosystem, evolving to support new use cases and the ever growing richness of the Kubernetes ecosystem."
2. We could add a little more context to introducing SSA - make clear what's new, make clear how SSA helps to support the needs articulated previously, note how SSA is somewhat new in Kubernetes, and that this being available broadly in the ecosystem now makes this a great time to start benefiting from it?

Reply:
Opened https://github.com/pulumi/pulumi-hugo/pull/1786 with some additional content